routers/repo/branch.go

```diff
@@ -29,3 +30,21 @@ func Branches(ctx *context.Context) {
 	ctx.Data["Branches"] = brs
 	ctx.HTML(200, BRANCH)
 }
+
+func DeleteBranchPost(ctx *context.Context) {
+	if !ctx.Repo.IsWriter() {
+		ctx.Error(404)
+		return
+	}
+
+	branchName := ctx.Params(":name")
+
+	if err := ctx.Repo.GitRepo.DeleteBranch(branchName, git.DeleteBranchOptions{
+		Force: false,
+	}); err != nil {
+		ctx.Handle(500, "DeleteBranch", err)
+		return
+	}
+
+	ctx.Redirect(ctx.Repo.RepoLink)
```

Comment:
Should this route receive route to redirect to?

I decided to redirect back to repo page because it may be call from some where else in the future. So, redirect back to PR page is still not correct.

What do you think?